app/page/send/index.tsx

```diff
@@ -0,0 +1,213 @@
+}
+
+const SendAmount = ({ amount, onChangeAmount }) => {
+  const [invalidValue, setInvalidValue] = useState<boolean>(false)
+
+  const handleAmountChange = async (event: ChangeEvent<HTMLInputElement>) => {
+    const value = event.target.value
+    onChangeAmount(value)
+    try {
+      console.log(`${ethers.parseUnits(value, "ether")}`)
+      setInvalidValue(false)
+    } catch (error) {
+      console.log(`Invalid value`)
+      setInvalidValue(true)
+    }
+  }
+
+  return (
+    <div className="flex">
+      <label className="flex-1">Amount:</label>
+      <input
+        type="text"
+        id="amount"
+        value={`${amount}`}
+        onChange={handleAmountChange}
+        className={`border w-96 outline-none ${
+          invalidValue ? "border-red-500" : "border-gray-300"
+        }`}></input>
+    </div>
+  )
+}
+// Select token -> Select address -> Send amount -> Review
+
+export function Send() {
+  const { provider } = useProviderContext()
+  const account = useAccount()
+  const nativeToken: Token = {
+    address: "0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE",
+    symbol: `${getChainName(account.chainId)}ETH`,
+    decimals: 18,
+    balance: account.balance
+  }
+  const tokens = [nativeToken, ...useTokens()]
+  const [token, setToken] = useState<Token>(tokens[0])
+  const [txTo, setTxTo] = useState<HexString>("")
+  const [txValue, setTxValue] = useState<BigNumberish>("0")
+  const [step, setStep] = useState<number>(0)
+
+  const stepsComponents = [
+    <SelectToken key="step1" tokens={tokens} onSelectToken={setToken} />,
+    <SelectAddress key="step2" to={txTo} onChangeTo={setTxTo} />,
+    <SendAmount key="step3" amount={txValue} onChangeAmount={setTxValue} />
+  ]
+  const stepsTitle = ["Select Token", "Select Address", "Send Amount"]
+
+  const handlePrevStep = useCallback(() => {
+    if (step > 0) {
+      setStep(step - 1)
+    }
+  }, [step])
+
+  const handleNextStep = useCallback(() => {
+    if (step < 2) {
+      setStep(step + 1)
+    }
+  }, [step])
+
+  const handleSend = useCallback(async () => {
+    const signer = await provider.getSigner()
+    if (address.isEqual(token.address, nativeToken.address)) {
+      return sendNativeToken(signer, txTo, txValue)
+    }
+    return sendErc20Token(signer, txTo, txValue, token)
+  }, [txTo, txValue])
+
+  return (
+    <>
+      {step === 0 ? (
+        <StepBackHeader title={stepsTitle[0]} href={Path.Index} />
+      ) : (
+        <StepBackHeader
+          title={stepsTitle[step]}
+          handleOnClick={handlePrevStep}
+        />
+      )}
+      {stepsComponents[step]}
+      <div className="flex">
+        {step < 2 ? (
+          <button onClick={handleNextStep} className="flex-1">
+            Next
+          </button>
+        ) : (
+          <>
+            <Link href={Path.Home} className="flex-1">
+              Cancel
+            </Link>
+            <button onClick={handleSend} className="flex-1">
+              Send
+            </button>
```

Comment:
Q: To my understanding we don't have a last step to send, right? When the last next is clicked, extension would redirect to review page for authorization.

Reply:
Yes, you're right. 
I remove this in the next PR #168